java/com/google/security/wycheproof/testcases/EciesTest.java

```diff
@@ -296,7 +299,7 @@ public void testIsAlias(String algorithmA, String algorithmB) throws Exception {
     byte[] message = "Hello".getBytes("UTF-8");
     eciesA.init(Cipher.ENCRYPT_MODE, keyPair.getPublic());
     byte[] ciphertext = eciesA.doFinal(message);
-    eciesB.init(Cipher.DECRYPT_MODE, keyPair.getPrivate());
+    eciesB.init(Cipher.DECRYPT_MODE, keyPair.getPrivate(), eciesA.getParameters());
```

Comment:
@thaidn Sorry for the delayed reply. Actually eciesA.getParameters() was correct, since we are trying to decrypt with the parameters that were used (possibly auto-generated) during encryption. Looking at current code it appears this was actually changed to eciesB, which ought to be reverted (there's currently a test failure with latest BC caused by using eciesB here).